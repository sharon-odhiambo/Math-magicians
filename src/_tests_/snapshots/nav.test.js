import React from 'react';
import {
  BrowserRouter as Router,
} from 'react-router-dom';
import { render } from '@testing-library/react';
import Nav from '../../spa/nav.js';

describe('Nav', () => {
  test('should match Nav snapshot', () => {
    const tree = render(
      <Router>
        <Nav />
      </Router>,
    );
    expect(tree).toMatchSnapshot();
  });
});
