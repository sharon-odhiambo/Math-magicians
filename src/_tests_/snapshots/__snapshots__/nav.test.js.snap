// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match Nav snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <h1>
          Math Magicians
        </h1>
        <nav>
          <a
            href="/"
          >
            Home
          </a>
          <a
            href="/calculator"
          >
            Calculator
          </a>
          <a
            href="/quote"
          >
            Quote
          </a>
        </nav>
      </header>
      <div
        class="home"
      >
        <h2>
          Welcome to my Page!
        </h2>
        <p>
          This page allows you to access the Calculator through the navigation. You can use the calculator to have some maths fun. Let's have fun!
          <br />
          Some quote for you too!
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <header>
      <h1>
        Math Magicians
      </h1>
      <nav>
        <a
          href="/"
        >
          Home
        </a>
        <a
          href="/calculator"
        >
          Calculator
        </a>
        <a
          href="/quote"
        >
          Quote
        </a>
      </nav>
    </header>
    <div
      class="home"
    >
      <h2>
        Welcome to my Page!
      </h2>
      <p>
        This page allows you to access the Calculator through the navigation. You can use the calculator to have some maths fun. Let's have fun!
        <br />
        Some quote for you too!
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
